locations/fblipke.yml

```diff
@@ -0,0 +1,152 @@
+contacts:
+  - 'contact@fblipke.de'
+  - '@lipkegu:matrix.org'
+hosts:
+# Proxmox VM (QEMU)
+  - hostname: fblipke-core
+    role: corerouter
+    model: "x86-64"
+    host__packages__to_merge:
+      - "nano qemu-ga kmod-brcmfmac kmod-brcmutil brcmfmac-firmware-4366c0-pcie"
+    wireless_profile: fblipke
+    wifi_roaming: true
+
+#  NSM5
+  - hostname: fblipke-ap01
+    role: ap
+    model: "ubnt_nanostation_m5_xw"
+    wireless_profile: fblipke
+    wifi_roaming: true
+
+# NETWRORKS:
+# Assigned: 10.248.12.0/24; 2001:bf7:850:1b00::/56
+
+# DHCP: 10.248.12.128/25
+# MGMT: 10.248.12.0/26
+# MESH: 10.248.12.64/26
+
+ipv6_prefix: "2001:bf7:850:1b00::/56"
+
+dhcp_no_ping: false
+
+networks:
+  - vid: 53
+    role: mesh
+    name: mesh_5g
+    prefix: 10.248.12.64/32
+    ipv6_subprefix: -20
+    mesh_ap: fblipke-ap01
+    mesh_radio: 11a_standard
+    mesh_iface: mesh
+
+  - vid: 52
+    role: dhcp
+    prefix: 10.248.12.128/25
+    ipv6_subprefix: 0
+    inbound_filtering: true
+    enforce_client_isolation: true
+    assignments:
+      fblipke-gw: 4
+      fblipke-core: 1
+
+  - vid: 42
+    role: mgmt
+    prefix: 10.248.12.0/26
+    gateway: 1
+    ntp: 1
+    dns: 1
+    ipv6_subprefix: 1
+    assignments:
+      fblipke-gw: 4
+      fblipke-core: 1 # QEMU - 10.248.12.1
+      fblipke-ap01: 2 # NSM5 - 10.248.12.2
+
+  - vid: 40 # WIFI (Private)
+    role: ext
+    name: lan40
+    no_corerouter_dns_record: true
+    enforce_client_isolation: false
+
+  - vid: 51
+    role: uplink
+
+  - role: tunnel
+    ifname: ts_wg0
+    mtu: 1280
+    prefix: 10.248.12.65/32
+    wireguard_port: 51820
+
+  - role: tunnel
+    ifname: ts_wg1
+    mtu: 1280
+    prefix: 10.248.12.66/32
+    wireguard_port: 51821
+
+location__wireless_profiles__to_merge:
+  - name: fblipke
+    devices:
+      - radio: 11g_standard
+        disabled: false
+
+      - radio: 11a_mesh
+        disabled: false
+
+      - radio: 11a_standard
+        disabled: false
+
+    ifaces:
+      - mode: ap
+        ssid: berlin.freifunk.net
+        encryption: none
+        network: dhcp
+        radio: [11a_standard, 11g_standard]
+        ifname_hint: ff
+        owe_transition_ifname_hint: ffowe
+
+      - mode: ap
+        ssid: Lipke-Intern
+        encryption: none # Captive Portal
+        network: lan40
+        radio: [11a_standard, 11g_standard]
+        ifname_hint: pr
+
+      - mode: mesh
+        mesh_id: Mesh-Freifunk-Berlin
+        radio: [11a_standard, 11g_standard, 11a_mesh]
+        mcast_rate: 12000
+        mesh_fwding: 0
+        ifname_hint: mesh
+
+      - mode: ap
+        ssid: berlin.freifunk.net OWE
+        hidden: true
+        encryption: owe
+        network: dhcp
+        radio: [11a_standard, 11g_standard]
+        ifname_hint: ffowe
+        owe_transition_ifname_hint: ff
+        ieee80211w: 1
+
+# AP-id, wifi-channel, bandwidth, txpower
+location__channel_assignments_11a_standard__to_merge:
+  fblipke-core: 36-80
+  fblipke-ap01: 52-80
```

Comment:
The Nanostation M5 only supports 40 MHz channels. In addition, if you want it to mesh with default Falter installs in your area the channel needs to be 36.